drp.py

```diff
@@ -457,7 +457,7 @@ def on_modified_async(self, view):
                 logger.info("Setting presence to file %r from %r", view.file_name(), last_file)
                 handle_activity(view)
 
-    def on_load_async(self, view):
+    def on_activated_async(self, view):
         handle_activity(view)
```

Comment:
Actually, I just realized that there is also `on_modified_async` which listens for changes and only then sets the active view (if the previous didn't change), so I think this will need some more consideration as to what the goal of the package is.

Should it only announce views that you made changes in or should it announce views that you focused? Depending on which we choose, we'd need to define either `on_modified_async` or `on_activated_async` but not both.

Either way, we'd also need a check for the `view.element()` so that we don't handle widget inputs such as the find panel or the command palette because those invoke `on_activated`.

Reply:
Based on the fact it had on_loaded and on_modified it would seem they are trying to capture both. However, the default message is "Editing in {}"

I think it would make sense to limit it to on_modified only. 

@Snazzah thoughts?.

Reply:
> Should it only announce views that you made changes in or should it announce views that you focused?
Should announce views you focused on. Whatever the user is mainly looking at should be displayed through the presence.